libraries/AP_NavEKF2/AP_NavEKF2_Control.cpp

```diff
@@ -265,7 +265,7 @@ void NavEKF2_core::setAidingMode()
         switch (PV_AidingMode) {
         case AID_NONE:
             // We have ceased aiding
-            gcs().send_text(MAV_SEVERITY_WARNING, "EKF2 IMU%u has stopped aiding",(unsigned)imu_index);
+            gcs().send_text(MAV_SEVERITY_WARNING, "EKF2 %u (g%u a%u) has stopped aiding",(unsigned)core_index, (unsigned)gyro_index, (unsigned)accel_index);
```

Comment:
I don't think this change in msg really helps - aiding isn't about IMU indexes

Reply:
My preference is that all the aiding related messages only print the EKF core index and that notification of a change in a core to use of a different IMU is left to be handled by the switchGyros(...) and switchAccels(...) functions